ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/filter/LogsQueryTest.java

```diff
@@ -428,23 +426,4 @@ public void multivariateTopicMatchMultivariateQueryReturnTrue() {
     assertThat(query.couldMatch(LogsBloomFilter.builder().insertLog(log).build())).isTrue();
     assertThat(query.matches(log)).isTrue();
   }
-
-  @Test
-  public void emptySubTopicProducesNoMatches() {
-    final Address address = Address.fromHexString("0x1111111111111111111111111111111111111111");
-
-    final LogTopic topic1 =
-        LogTopic.fromHexString(
-            "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa");
-    final LogTopic topic2 =
-        LogTopic.fromHexString(
-            "0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb");
-    final List<List<LogTopic>> queryParameter =
-        Lists.newArrayList(singletonList(topic1), emptyList());
-
-    final LogsQuery query = new LogsQuery.Builder().address(address).topics(queryParameter).build();
-    final Log log = new Log(address, data, Lists.newArrayList(topic1, topic2));
-
-    assertThat(query.matches(log)).isFalse();
-  }
```

Comment:
note there are 2 LogsQueryTest classes - I think the other one has clearer structure so I added a test there and removed this one

Reply:
prob should combine them - in a separate PR